docs/rules/no-static-imports-of-user-specific-scoped-modules-in-ssrable-components.md

```diff
@@ -0,0 +1,62 @@
+# Deprecation Notice for Salesforce Scoped Modules during Server-Side Rendering (SSR)
+
+Static imports of user-specific scoped modules, such as `@salesforce/user/*`, are not supported in LWC components marked with `lightning__ServerRenderable` or `lightning__ServerRenderableWithHydration`.
+
+## Rule details
+
+The following Salesforce scoped modules are deprecated when using Server-Side Rendering (SSR):
+
+-   `@salesforce/user/*`
+-   `@salesforce/userPermission/*`
+-   `@salesforce/customPermission/*`
+-   `@salesforce/featureFlag/*`
```

Comment:
`@salesforce/featureFlag` is internal, can we just not mention it here?

Reply:
Oh, good to know. @lpomerleau , FYI, I'm going to remove it from the rule.

Reply:
`featureFlag` is private and should be removed from this public PR.